recipes/sphinx-autobuild/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: sphinx-autobuild
+  version: "0.6.0"
+
+source:
+  fn: sphinx-autobuild-0.6.0.tar.gz
+  url: https://pypi.python.org/packages/85/cf/25b65781e6d2a4a89a431260daf1e0d53a81c52d27c98245481d46f3df2a/sphinx-autobuild-0.6.0.tar.gz
```

Comment:
Please use the canonical pypi url. For example:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18